charts/linkerd-smi/values.yaml

```diff
@@ -2,10 +2,9 @@ namespace: linkerd-smi
 
 installNamespace: true
 clusterDomain: cluster.local
-linkerdSMIVersion: &linkerd_smi_version linkerdSMIVersionValue
 
 adaptor:
   image:
     name: cr.l5d.io/linkerd/smi-adaptor
-    version: *linkerd_smi_version
+    version: linkerdSMIVersionValue
```

Comment:
Are you planning on including something like `bin/helm-build` to replace this placeholder and to package the chart, in a separate release PR?

Reply:
Yeah, That will be part of the release PR! Didn't add it as it is not needed for the integration tests! :)